lib/elixir_sso_example_web/templates/page/index.html.eex

```diff
@@ -1,9 +1,9 @@
-<%= if assigns[:profile] do %>
-  Welcome <%= @profile["first_name"] %>
-<% else %>
-  <h1>Welcome to WorkOS Elixir!</h1>
-  <%= if assigns[:error] do %>
-    <div class="error">Error <%= @error %></div>
-  <% end %>
-  <%= link "Click Here to Login", to: Routes.page_path(@conn, :login), class: "button" %>
-<% end %>
+<body class="container_login">
+        <div class='flex_column'>
```

Comment:
The interpolation is a little bit weird here, maybe we could try to configure Prettier to format those templates 

Ref: https://odysee.com/@slymilano:2/phoenix-liveview-template-formating-automatic:2?r=f98nZqd8KjewJn3jMYr2PggLCNinNKnv